installer/IL15_emulated_checks_init.sh

```diff
@@ -85,8 +85,7 @@ IL15_emulated_checks_init() {
       # EMBAbite fuzzer used this:
       # pip_install "upnpclient"
       pip_install "lxml==5.3.1"
-      pip_install "upnpclient" "--no-dependencies lxml"
-
+      pip3 install "upnpclient" --no-dependencies lxml
```

Comment:
Why pip directly 
Would add comment

Reply:
good point. Our current pip_install helper function does not support further parameters.

Reply:
done